implant/sliver/handlers/rpc-handlers.go

```diff
@@ -128,6 +127,25 @@ func ifconfig() *sliverpb.Ifconfig {
 		}
 		interfaces.NetInterfaces = append(interfaces.NetInterfaces, netIface)
 	}
+}
+
+func ifconfig() *sliverpb.Ifconfig {
+	netInterfaces, err := net.Interfaces()
+	if err != nil {
+		return nil
+	}
+
+	interfaces := &sliverpb.Ifconfig{
+		NetInterfaces: []*sliverpb.NetInterface{},
+	}
+
+	ifconfigParseInterfaces(netInterfaces, interfaces)
+
+	// Linux namespace handling
+	if runtime.GOOS == "linux" {
```

Comment:
That won't work (as shown in the CI errors).
You basically need to make a dedicated` sub-package with platform specific code in it so you can call it from `rpc-handlers`.

Reply:
Thanks! I'll work on getting that fixed up as well :)